pkg/operation/botanist/component/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -411,7 +411,13 @@ func (k *kubeControllerManager) Deploy(ctx context.Context) error {
 		}
 
 		if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, k.seedClient.Client(), hvpa, func() error {
-			hvpa.Labels = getLabels()
+			hvpa.Labels = utils.MergeStringMaps(
+				hvpa.GetLabels(),
```

Comment:
Perhaps using `hvpa.Lables` would prevent confusion?
```suggestion
				hvpa.Labels,
```